### PR TITLE
Support disclaimer and preview install directions

### PR DESCRIPTION
**Pull request by jnoller**
Updated to note support limits, added instructions and caveats on the CLI install.

### CODE REVIEW

File: previews.md [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
+az feature register -n VMSSPreview --namespace Microsoft.ContainerService
+```
+
+Then refresh your registration of the AKS resource provider:
+
+```
+az provider register -n Microsoft.ContainerService
+```
+
+To create a cluster with network policy enabled, use the `--network-policy` switch in `az aks create` and specify the network policy plugin you would like to use, `azure` or `calico`. Note that both require the `azure` network plugin.
```
Comment (jluk) [inline previews.md]: Should this be under the below header "Kubernetes Network Policy"? Seems odd its under the VMSS header.

File: previews.md [hunk 2]
```diff
@@ -1,13 +1,48 @@
...
+In order to use / opt into preview features, you will need to use the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) and ensure it is up to date with the latest release. Once that is complete, you must install the `aks-preview` extension via:
+
+```
+az extension add --name aks-preview
+```
+
+**Warning**: Installing the preview extension will update the CLI to use the **latest** extensions and API. Please do not enable this for production clusters. In order to uninstall the extension, you can do the following:
+
+```
+az extension remove --name aks-preview
```
Comment (jluk) [inline previews.md]: I think there's a misconception about "preview" APIs, the CLI extension should be using the same API as the prod one. Perhaps we should rephrase as "do not use preview properties within the API on production clusters"? We have the Swagger details for preview properties noted.

Comment (jluk) [reply previews.md]: FWIW I still have an action to get a definitive answer on how API versions are handled between cli extension and official cli cut. Something got handled in a very weird way during the VMSS release that split API versions that I am tracking down today.

File: previews.md (hunk 1 above)
Comment (jnoller) [reply previews.md]: sigh, copy paste error

**Pull request merged**